src/main/resources/project/procedures/decommissionEnvironments/tearDownEnvironments.pl

```diff
@@ -0,0 +1,137 @@
+#-------------------------------------------------------------------------
```

Comment:
ah never mind. I had assumed the dsl script had the procedure logic itself in either dsl or groovy. Apparently that is still in perl which is ok.